build_container/build_container_ubuntu.sh

```diff
@@ -65,6 +72,18 @@ case $ARCH in
           -o /usr/local/bin/bazel
         chmod +x /usr/local/bin/bazel
         ;;
+    'aarch64' )
```

Comment:
Hi, @lizan 
As far as I known, the bazelisk still does not support the arm64 platform. So i choose the install the Bazel from Ubuntu [PPA](https://launchpad.net/~kingofcodecplusplus/+archive/ubuntu/bazel120)

Reply:
If you point bazelisk to a fork hosting arm64 binaries it should work? bazlisk itself is written in Go so you can build it, no?

Reply:
Hi, lizan
The bazelisk downloads the bazel binary file from github release. But, unfortunately, there is no arm64 version in bazel release. So I have to choose installing bazel from PPA.

Reply:
Yes, I know that bazel official release doesn't have arm64 version, but can't you host the arm64 version in a fork, and point bazelisk to there? That's what https://github.com/bazelbuild/bazelisk#how-does-bazelisk-know-which-bazel-version-to-run-and-where-to-get-it-from is for. I'm asking this because we can't have specific version of bazel if we install from PPA.

Reply:
Did you mean i maintain a personal bazelisk and bazel repository. Bazelisk point to my personal bazel arm64 release. By my bazelisk, we could choose different bazel arm64 versions?

Reply:
Hi @lizan 
I know install bazel with specific version from PPA is not a good idea. But i think it is better than referring it to a personal Github repository.
On the other hand, I can not upload any binaries since company security policies. 
So, based on those 2 reasons, i choose the PPA.

Reply:
You don't have to specify where it points to in this PR, you only need when you actually do the build. So this gives more flexibility than PPA.

Reply:
I have updated the patch. Please review it.

Reply:
@lizan 
Hi, I have updated the patch. Could you please check it?